### DISCUSSION TITLE
Add quality filter to album view

### DISCUSSION THREAD

**Discussion by duenni** [navidrome/navidrome#2311] [Ideas]
Hi, 

thanks for this great software. 👏 
Would it be possible to add a 'quality' filter to the Album view? I would then be able to filter for e.g. all albums which are 128kbps MP3 or all FLAC albums. It could maybe also be a filter based on kbps alone. Something like: lossless, >=320kpbs, >=256kbps, >=192kbps, etc.
Thanks for considering.

Cheers

**Comment by deluan** [comment]
Yes, I'm rewriting the UI and will add more filters to it. Should be ready in the coming months.

**Comment by wie-was** [comment]
Since upgrading from 0.54.5 to 0.59, I noticed some new filters available, which is super cool! 
I'd love to be also able to filter out my high-res albums though. The best parameter in my opinion would be either "Bit rate" (higher than 16bit) or "Sample rate" (higher than 44.1kHz) or both conditions applied at the same time. Both parameters would work for me on their own, because the minimum resolution of all high-res audio I've encountered so far is 24bit/48kHz.

Other options that I've considered to sort my high-res music are:
* Writing a custom genre, eg. "High-Res", into the music file's tags: This works beautifully, but it's tedious to manually tag every new album.
* Using the multi-library feature: Having a separate folder for all high-res content. This works fine, but if you auto-tag and import music with Beets, Beets cannot filter out high-res albums and put them in a separate folder, as far as I've checked.
* Smart-Playlist: This works, but it tosses all tracks into a large playlists, with no album-overview.

**Reply by wie-was** [reply]
As an update, to whomever may be interested: For now, I've decided to write "High-Res Audio" into the "grouping" tag for albums with bit depth of 24 bit. Like that, I can use the built-in "Grouping" filter in navidrome. Some notes:
* In my collection, the "grouping" tag is not otherwise used.
* I'm modifying the meta tags of my high-res albums using the following beets command: `beet modify -y bitdepth:24 grouping="High-Res Audio"`
* I might run this command regularly using cron to keep the Grouping reasonably up to date.
